src/libraries/Microsoft.PowerFx.Core/Public/CheckResult.cs

```diff
@@ -25,7 +25,13 @@ public class CheckResult
         /// null if unavailable.  
         /// This is only valid if there are no errors. 
         /// </summary>
-        public HashSet<string> TopLevelIdentifiers { get; set; }
+        public HashSet<string> TopLevelIdentifiers { get; set; }
+
+        /// <summary>
+        /// Spans of all fields that this formula uses. 
```

Comment:
>Spans of all fields that this formula uses.  [](http://example.com/codeflow?start=12&length=44)

Can we clarify the difference between this and TopLevelIdentifiers?

Ie,  what does this do for:
 A.B.C? 
First(A.B).X.Y

Reply:
For the first line, it returns the span covering "A.B.C".  For the second, it returns the span covering "A.B".  Let's talk (outside the PR review?) about whether that is appropriate or not. I can see arguments on both sides.